### PR TITLE
6.7wip

### PR DESCRIPTION
**Pull request by jeffreytoffoli**

### CODE REVIEW

File: api-guide/model/model-types.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+Clarifai offers a wide variety of models that can be used as standalone solutions, or as the building blocks for your own custom business solution.
+
+This page describes some important model types that you should know when working with the Clarifai platform. Please keep in mind that this is an overview of the general categories of models available to you and that new models are created all the time. For a listing of the models available to you:
+
+* Make a `GET` call to `/models/types` api. The api responds with all available model types.
+* You can then choose any one of the model types and make a `POST` request to the `/models` api with the payload specified in the response for the chosen model type.
+
+## Trainable Models
+
+### Classification
```
Comment (yangshengBE) [inline api-guide/model/model-types.md:12]: should be visual-classifier? (also all other model types after)

i think we should make the name consistent with what we show in model mode.

Comment (jeffreytoffoli) [reply api-guide/model/model-types.md:12]: Aren't the visual-* models just for deep training?

Comment (yangshengBE) [reply api-guide/model/model-types.md:12]: that is correct. 
if you mean custom model, then it should be `Context-Based Classifier`

Comment (yangshengBE) [reply api-guide/model/model-types.md:12]: to be clear are these supposed to be model type name?

Comment (jeffreytoffoli) [reply api-guide/model/model-types.md]: ok i've made some updates to the structure here

File: api-guide/walkthroughs/aws-lambda.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
+# AWS Lambda Functions with Clarifai
+
+AWS Lambda
+
+## Accessing the clarifai_grpc package
+
+We need to setup some initial helper functions for users to get easy access to our clarifai_grpc package.
```
Comment (yangshengBE) [inline api-guide/walkthroughs/aws-lambda.md:7]: i think we need to document what are the req we will send to lambda function and what is the response we expect.
the request we will send is PostModelsOutput with previous models' output
expecting structured response like:
```
type AWSLambdaResponse struct {
	StatusCode int64                    `json:"statusCode"`
	Body       string                   `json:"body"`
	Response   *api.MultiOutputResponse `json:"response"`
}
```

and status code should be 200

File: api-guide/walkthroughs/aws-lambda.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+aws lambda add-layer-version-permission --layer-name clarifaiGRPCWithPythonFolder  --version-number 4  --statement-id allAccountsExample --principal="*"   --action lambda:GetLayerVersion
+
+```
+{% endtab %}
+{% endtabs %}
+
+
+## Instructions for Python Users
+
```
Comment (yangshengBE) [inline api-guide/walkthroughs/aws-lambda.md:58]: i think we need to document what are the req we will send to lambda function and what is the response we expect.
the request we will send is PostModelsOutput with previous models' output
expecting structured response like:

type AWSLambdaResponse struct {
	StatusCode int64                    `json:"statusCode"`
	Body       string                   `json:"body"`
	Response   *api.MultiOutputResponse `json:"response"`
}
and status code should be 200

File: api-guide/walkthroughs/aws-lambda.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+```
+{% endtab %}
+{% endtabs %}
+
+We will send a `PostModelsOuptut` call to the lambda function and the data payload you should expect is `PostModelsOutputRequest` with the previous model's output in workflow. The response will look like this:
+
+{% tabs %}
+{% tab title="gRPC Python" %}
```
Comment (yangshengBE) [inline api-guide/walkthroughs/aws-lambda.md:152]: actually this is not python but go struct.

File: api-guide/walkthroughs/aws-lambda.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+type AWSLambdaResponse struct {
+	StatusCode int64                    `json:"statusCode"`
+	Body       string                   `json:"body"`
+	Response   *api.MultiOutputResponse `json:"response"`
+}
+```
+{% endtab %}
+{% endtabs %}
+
+In the `statusCode` field you should expect a value of 200 if the lambda call is successful. In the `response` field you can expect a response in the form of a json object from `MultiOutputResponse`. We will then send the data returned from this object to the next model in your workflow.
```
Comment (yangshengBE) [inline api-guide/walkthroughs/aws-lambda.md:164]: replace `you` with `we`? 
we are expecting 200 response sent from user.

**Pull request merged**